src/http/structs.rs

```diff
@@ -100,9 +100,24 @@ pub struct EntityPath {
     pub entity: String,
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+pub struct EntityProfileLink {
+    pub url: String,
+    pub verified: bool,
+}
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct EntityProfile {
+    pub display_name: Option<String>,
+    pub description: Option<String>,
+    pub workplace: Option<String>,
+    pub links: Vec<EntityProfileLink>,
+}
+
 #[derive(Serialize, Deserialize, Clone)]
```

Comment:
I don't have a fully formed opinion, but I'm curious if we want to have a separation between `Entity` and `EntityQueryResponse` as you have in this PR. Are we separating them because we believe you should have to explicitly request this metadata separate from stuff like the type of entity? Is that for performance reasons or usability reasons?

Reply:
`Entity` doesn't contain things like the databases owned by the entity, that's why it's separated :)

Reply:
Owned databases makes sense to me, I'm more curious about the fields directly on the Entity struct/table. I'm good either way, and I get the sense your bias is toward separating them, which is OK with me. Just wanted to sanity-check! :)

Reply:
Alright, I'll keep them separated (at least for now). You've made me realize that the entity retrieval endpoint doesn't send the entity type. I'll change that in a future PR :)